site/docs/guides/flowctl/create-derivation.md

```diff
@@ -125,65 +78,49 @@ Then, you'll need to add your source collection's specification files to a **dra
+   Your SQL statements are evaluated with each source collection document. [Here](https://docs.estuary.dev/concepts/derivations/#sqlite) is an example of what the output document from a derivation would look like given an input document and [SQL lambda](https://docs.estuary.dev/concepts/derivations/#sql-lambdas).
 
-   * The collection must have a schema that reflects your desired transformation output.
-     They can be whatever you want, as long as they follow Flow's standard formatting.
-     For help, see the [schemas](../../concepts/schemas.md) and [collection key](../../concepts/collections.md#keys) documentation.
-   * Add the `derivation` stanza. The TypeScript module you name will be generated next, and you'll define the transformation's function there.
+## Preview the derivation
 
-   ```yaml
-   collections:
-      namespace/data-flow-name/greetings:
-        {...}
-      #The name for your new collection can be whatever you want,
-      #so long as you have permissions in the namespace.
-      #Typically, you'll want to simply copy the source prefix
-      #and add a unique collection name.
-      namespace/data-flow-name/dozen-greetings:
-         #In this example, our objective is to round the number of greetings to the nearest dozen.
-         #We keep the `count` and `message` properties from the source,
-         #and add a new field called `dozens`.
-         schema:
-            properties:
-               count:
-                  type: integer
-               message:
-                  type: string
-               dozens:
-                  type: integer
-            required:
-            - dozens
-            - count
-            - message
-            type: object
-         #Since we're interested in estimating by the dozen, we make `dozens` our collection key.
-         key:
-            - /dozens
-         derivation:
-            transform:
-               #The transform name can be anything you'd like.
-               greetings-by-dozen:
-                  #Paste the full name of the source collection.
-                  source: {name: namespace/data-flow-name/greetings}
-                  #This simple transform only requires a **publish lambda* function.
-                  #More complex transforms also use **update lambdas**.
-                  #See the Derivations documentation to learn more about lambdas.
-                  publish: {lambda: typescript}
-            #The name you provide for the module will be generated next.
-            typescript: {module: divide-by-twelve.ts}
-      ```
-   Save the file.
+Type this command on the Terminal tab in GitPod to **preview** the derivation.
+
+```console
+flowctl preview --source flow.yaml --interval 200ms | jq -c 'del(._meta)'
+```
+
+## Publish the derivation from GitPod
+
+**Publish** the catalog.
+
+```console
+flowctl catalog publish --source flow.yaml
+```
+
+## Updating an existing derivation
+
+SQL statements are applied on a go-forward basis only.
```

Comment:
You can say this about derivations generally, not just SQL.

Can this section be more agnostic to the type of derivation?